service/header/store_heightsub.go

```diff
@@ -20,16 +20,21 @@ func newHeightSub() *heightSub {
 	}
 }
 
+// Height reports current height.
+func (hs *heightSub) Height() uint64 {
```

Comment:
Can we call it `LatestAvailableHeight` or something? I'm in favour of higher verbosity.

Reply:
I don't see where is the boundary between `LatestAvailableHeight` and `LatestAvailableHeightOfSubjectiveCelestiaDABlockchain` tbh

Reply:
`HeadHeight`as compromise?

Reply:
Refer to: 
https://github.com/celestiaorg/celestia-node/pull/428#discussion_r805345176

Reply:
ack